website/content/en/v0.20.0/faq.md

```diff
@@ -90,6 +90,27 @@ See [Node NotReady]({{< ref "./troubleshooting/#node-notready" >}}) troubleshoot
 
 Though not recommended, if you need to access Karpenter-managed nodes without AWS credentials, you can add SSH keys using AWSNodeTemplate.
 See [Custom User Data]({{< ref "./concepts/operating-systems/" >}}) for details.
+Though not recommended, if you need to access Karpenter-managed nodes without AWS credentials, you can add SSH keys to Launch Templates through [Custom User Data](https://karpenter.sh/preview/aws/user-data/).
```

Comment:
looks like this was duplicated or copied incorrectly?

Reply:
indeed, maybe came from some UI-done rebase